### PR TITLE
Remove references to the telephony gem.

### PR DESCRIPTION
**Pull request by kevindayton**

### CODE REVIEW

File: _articles/github.md [hunk 1]
```diff
@@ -29,9 +29,6 @@ View a [list of all repositories](https://github.com/topics/login-gov) tagged fo
 
 ### Libraries
 
-- [**18f/identity-telephony**](https://github.com/18f/identity-telephony)
-  Gem that provides a common interface for voice and SMS features across multiple vendor backends
-
```
Comment (zachmargolis) [inline _articles/github.md:34]: no need to remove! See the example of marking gems as deprecated below with the `{: .deprecated-link} (Archived)` and comments like "This code has been moved into the IDP codebase."

Comment (kevindayton) [reply _articles/github.md:34]: Got it.  Push this change.

**Review by zachmargolis — APPROVED** [review]
LGTM! thanks!

File: _articles/github.md [hunk 2]
```diff
@@ -41,6 +38,9 @@ View a [list of all repositories](https://github.com/topics/login-gov) tagged fo
 - [**18f/identity-validations**](https://github.com/18f/identity-validations)
   Gem that provides shared validations for the ServiceProvider model across the IDP and dashboard
 
+- [**18f/identity-telephony**](https://github.com/18f/identity-telephony){: .deprecated-link} (Archived)
+Gem that provides a common interface for voice and SMS features across multiple vendor backends. This code has been moved into the IDP codebase.
```
Comment (zachmargolis) [inline _articles/github.md:42]: indent to match other lines?
```suggestion
  Gem that provides a common interface for voice and SMS features across multiple vendor backends. This code has been moved into the IDP codebase.
```

**Pull request merged**